app/components/Home.jsx

```diff
@@ -6,23 +6,35 @@ import bulb from './public/bulb.png';
 import controller from './public/controller.png';
 
 class Home extends PureComponent {
+    constructor(props){
+        super(props);
+        this._handleDropdownClick = this._handleDropdownClick.bind(this)
+        this.state = {
+            dropdownVisible: false
+        }
+    }
+
+    _handleDropdownClick() {
+        this.state.dropdownVisible === false ? this.setState({dropdownVisible: true}) : this.setState({dropdownVisible: false})
+    }
```

Comment:
This can be simplified .... below I passed true of false into the _handleDropdownClick method ... so this becomes just _handle(status) {this.setState({isDropdownVisible: status})